infra/terraform/application.tf

```diff
@@ -92,6 +106,13 @@ module "dev_application" {
   frontdoor_profile_uuid         = module.dev_frontdoor[0].resource_guid
   public_network_access_enabled  = true
 
+  identity = {
+    type         = "SystemAssigned, UserAssigned"
```

Comment:
does this create both a systemassigned and userassigned? do you need both?

Reply:
Yes, this will create both.  We need both as Redis is the first use case that uses user assigned managed identity.  Key Vault still uses the System Assigned managed identity.  We have an item in the backlog to transition to user assigned managed identity.